Sources/Tracking/CustomerIO.swift

```diff
@@ -230,6 +234,17 @@ public class CustomerIO {
             }
         }
     }
+    
+    /**
+     Set the active identifier without calling the customer.io API
+    **/
```

Comment:
Why?

Reply:
was to get around some test stuff I was trying to work with. found an alt approach & removed it